doc/source/parameters/cfd/volume_of_fluid.rst

```diff
@@ -0,0 +1,203 @@
+
+  * ``enable``: controls if ``surface tension force`` is considered.
+  * ``verbosity``: enables the display of the output from the surface tension force calculations. Choices are: ``quiet`` (default, no output) and ``verbose``.
+  * ``output auxiliary fields``: enables the display of the filtered ``phase fraction gradient`` and filtered ``curvature``. Used for debugging purposes.
+  * ``surface tension coefficient``: surface tension coefficient in :math:`Nm^{-1}`.
+  * ``phase fraction gradient filter``: value used to apply a `projection step <https://onlinelibrary.wiley.com/doi/full/10.1002/fld.2643>`_ to damp high frequency errors, that are magnified by differentiation, in the phase fraction gradient (:math:`\bf{\psi}`), following the equation:
+
+    .. math::
+        \int_\Omega {\bf{v}} . {\bf{\psi}} + \eta_n \nabla {\bf{v}} . \nabla {\bf{\psi}} d\Omega = \int_\Omega {\bf{v}} . \nabla {\phi} d\Omega
+
+    where :math:`\bf{v}` is a piecewise continuous vector-valued test function, :math:`\bf{\psi}` is the filtered phase fraction gradient, :math:`\eta_n \geq 0` is the ``phase fraction gradient filter`` value, and :math:`\phi` is the phase fraction.
+
+  .. tip::
+
+    The ``phase fraction gradient filter`` must be a small value larger than 0. Use the procedure suggested in: :ref:`choosing values for the surface tension force filters`.
+
+  * ``curvature filter``: value used to apply a `projection step <https://onlinelibrary.wiley.com/doi/full/10.1002/fld.2643>`_ to damp high frequency errors, that are magnified by differentiation, in the curvature (:math:`k`), following the equation:
+
+    .. math:: 
+        \int_\Omega v k + \eta_k \nabla v . \nabla k d\Omega = \int_\Omega \nabla v . \frac{\bf{\psi}}{|\bf{\psi}|} d\Omega
+
+    where :math:`v` is a test function, :math:`k` is the filtered curvature, :math:`\eta_k` is the ``curvature filter`` value, and :math:`\bf{\psi}` is the filtered phase fraction gradient. 
+
+  .. tip::
+
+    Use the procedure suggested in: :ref:`choosing values for the surface tension force filters`.
+
+
+
+
+.. _improve wetting:
+
+Improving the Wetting mechanism
++++++++++++++++++++++++++++++++++++
+
+In the framework of incompressible fluids, a layer of the lowest density fluid (e.g. air) can form between the highest density fluid (e.g. water) and the boundary, preventing its wetting. Two strategies can be used to improve the wetting mechanism:
+
+1. add a small ``diffusion`` to the transport equation (e.g. ``set diffusion = 1e-3``), so that the higher density fluid spreads to the boundary location. 
+
+.. tip::
+  It is strongly advised to sharpen the interface more often (e.g. ``set sharpening frequency = 2``) to limit interface blurriness due the added diffusion. As peeling-wetting is handled after the transport equation is solved, but before interface sharpening, this will not prevent the wetting to occur.
```

Comment:
```suggestion
  It is strongly advised to sharpen the interface more often (e.g. ``set sharpening frequency = 2``) to limit interface blurriness due the added diffusion. As peeling-wetting is handled after the transport equation is solved, but before interface sharpening, this will not prevent the wetting from occuring.
```